src/environments/environment.prod.ts

```diff
@@ -1,20 +1,49 @@
+import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { AuthConfig } from 'angular-oauth2-oidc';
+import { TransactionsInterceptor } from '../app/interceptors/transactions.interceptor';
+import { AccountsInterceptor } from '../app/interceptors/accounts-interceptor';
+
+const pageConfig = {
+  locales: '${LOCALES}'.split(','),
```

Comment:
Isn't there a way to retrieve the list of locales from Angular injection tokens? Did we look at Angular CLI docs/issues? Would be a nice candidate for a PR to the Angular project!

Reply:
I will be checking if I can contribute. In the meantime I will leave this like that for the sake of stability and demo